addon/components/nrg-dropdown.js

```diff
@@ -380,7 +384,12 @@ export default class NrgDropdownComponent extends NrgValidationComponent {
     const clickedInsideDropdownElement = evt.target.closest('.dropdown');
     if (!this.search || isDropdownIcon) {
       if (this.isOpen && (isDropdownIcon || !clickedInsideDropdownElement)) {
-        this.isOpen = false;
+        if (
+          isDropdownIcon == clickedInsideDropdownElement ||
+          this.closeOnInternalSelect
```

Comment:
```suggestion
          this.closeOnInternalSelect ||
          isDropdownIcon == clickedInsideDropdownElement
```

If we reorder these conditions, we can make it so that we will only check the new requirement when `this.closeOnInternalSelect` is false. This should help ensure that we aren't changing any existing functionality, since we default that value to `true`, it will only check `isDropdownIcon == clickedInsideDropdownElement` when the consumer passes the options as `false`.